src/main/java/seedu/dukeofbooks/common/IVerifiable.java

```diff
@@ -0,0 +1,7 @@
+package seedu.dukeofbooks.common;
+
+public interface IVerifiable {
+    public abstract boolean equals(Object other);
+    public abstract int hashCode();
+    public abstract String toString();
```

Comment:
A minor issue: it is redundant to write `public` and `abstract` for methods in an interface. From [Oracle Java Documentation](https://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.4):  

> It is permitted, but discouraged as a matter of style, to redundantly specify the public and/or abstract modifier for a method declared in an interface.

You can directly write:
``` java
public interface IBlahBlah {
    int foo();
    void bar();
}

```